tests/integration/integration_tests.py

```diff
@@ -0,0 +1,155 @@
+import asyncio
+import json
+import os
+import re
+
+import numpy as np
+import requests
+import structlog
+import yaml
+from dotenv import find_dotenv, load_dotenv
+from sklearn.metrics.pairwise import cosine_similarity
+
+from codegate.inference.inference_engine import LlamaCppInferenceEngine
+
+logger = structlog.get_logger("codegate")
+
+
+class CodegateTestRunner:
+    def __init__(self):
+        self.inference_engine = LlamaCppInferenceEngine()
+        self.embedding_model = "./models/all-minilm-L6-v2-q5_k_m.gguf"
```

Comment:
since we already have the model at `codegate_volume/models/all-minilm-L6-v2-q5_k_m.gguf` in the repo why not point at that instead?

Reply:
Yes, fixed.